params/types/coregeth/chain_config_configurator.go

```diff
@@ -600,6 +603,12 @@ func (c *CoreGethChainConfig) SetEthashEIP649Transition(n *uint64) error {
 		return nil
 	}
 
+	if eip1234 := c.GetEthashEIP1234Transition(); eip1234 != nil {
+		if *eip1234 <= *n {
+			return nil
+		}
+	}
+
 	c.ensureExistingRewardSchedule()
 	c.BlockRewardSchedule[*n] = vars.EIP649FBlockReward
 
```

Comment:
These two changes are important. The prevent a configuration from overwriting an already-configured EIP1234 value with a same-block EIP649 configuration; essentially codifying that EIP649 can not be set after EIP1234.